docs/DeveloperGuide.md

```diff
@@ -290,27 +290,28 @@ Furthermore, it can provide easy categorisation and filtering of patients.
-| `* *`    | Silly user     | receive confirmation before deleting a contact               | I don’t accidentally delete important information            |
-| `* *`    | User           | sort the patients according to appointment dates             | I can easily know which are the latest upcoming appointments |
-| `* *`    | User           | add a contact to my favorite list                            | I can quickly access important contacts                      |
-| `* *`    | User           | see alerts of duplicate contacts in the app                  | I can keep the contact list clean and avoid redundancy and confusion |
-| `* *`    | User           | tag patients with recurring appointments                     | I know those who need regular follow ups                     |
-| `* *`    | Expert User    | categorize patients based on medical conditions, assigned doctor(s) and/or treatment plan | I can prioritize urgent conditions and streamline patient management. |
-| `* *`    | User           | add notes to a contact                                       | I can remember important information about that person.      |
-| `*`      | User           | filter patients based on appointment dates                   | I can contact and remind them.                               |
-| `*`      | Potential user | see the app populated with some sample commands              | I can easily learn how to use the app.                       |
-| `*`      | User           | use the app to work offline                                  | I can use it even when there is no internet connection.      |
+| `* * *`  | User           | save addresses                                                                   | have their location readily available.                                                     |
+| `* * *`  | User           | delete a contact                                                                 | remove outdated or incorrect contact information                                           |
+| `* * *`  | User           | list all contacts in one dashboard                                               | easily view all the contacts that I have added.                                            |
+| `* * *`  | User           | find the patient contact by a keyword (e.g. name, phone numbers)                 | search the patients’ contact instantly.                                                    |
+| `* *`    | User           | edit an existing patient contact                                                 | update their details when necessary.                                                       |
+| `* *`    | User           | see all the texts and UI clearly                                                 | don’t have to squint my eyes                                                               |
```

Comment:
For this line testers might point out that it's not exactly correct. Maybe we can replace it with 'avoid squinting my eyes' in order to follow the 'So that I can...' correctly.